pkg/apply/task/inv_add_task.go

```diff
@@ -94,7 +104,7 @@ func inventoryNamespaceInSet(inv inventory.Info, objs object.UnstructuredSet) *u
 	for _, obj := range objs {
 		gvk := obj.GetObjectKind().GroupVersionKind()
 		if gvk == namespaceGVKv1 && obj.GetName() == invNamespace {
-			inventory.AddInventoryIDAnnotation(obj, inv)
+			inventory.AddInventoryIDAnnotation(obj, inv.ID())
```

Comment:
huh. what is this call doing here? 

Seems weird of modify the object in a function called `inventoryNamespaceInSet`. That name makes it sound like it should return a bool.

Maybe it would be more clear if named `inventoryNamespaceFromSet`? But then, why are we modifying it in this function? Seems like something the caller should do and say why they're doing it.